source/localizable/csvspec/index.en.html.md.erb

```diff
@@ -206,7 +206,8 @@ photos of the same product from different angles or with a different
 presentation or combinations of the above add value to the presentation
 of the product in <%= settings.site_name.capitalize %>.
 
-This field is optional but recommended. It can occur multiple times
+This field is optional but recommended, except of fashion categories
```

Comment:
`except for` as @Zorbash mentioned below.

Reply:
fixed